package.json

```diff
@@ -180,6 +180,7 @@
     "isparta": "^4.0.0",
     "istanbul": "^0.4.4",
     "jasmine-reporters": "^2.2.0",
+    "jest": "^17.0.3",
```

Comment:
Unfortunately grunt-jest depends on it. Just pushed up an update to packages and snapshots for new version of jest